net-misc/networkmanager-fortisslvpn/files/networkmanager-fortisslvpn-1.2.2-location-fix.diff

```diff
@@ -0,0 +1,11 @@
+diff --git a/src/Makefile.am b/src/Makefile.am
```

Comment:
Is this patch upstreamed? What is its status? Also, please add some description what it fixes.